src/Persistence/Configuration/SaleConfiguration.cs

```diff
@@ -0,0 +1,22 @@
+﻿using Domain.Entity;
+using Microsoft.EntityFrameworkCore;
+using Microsoft.EntityFrameworkCore.Metadata.Builders;
+
+namespace Data.Configuration;
+
+public class SaleConfiguration : IEntityTypeConfiguration<SaleEntity>
+{
+    public void Configure(EntityTypeBuilder<SaleEntity> builder)
+    {
+            builder.HasKey(sale => sale.Id);
+            builder.HasOne(sale => sale.BillingParty)
+                .WithMany();
+            builder.OwnsMany(sale => sale.Sales,
+                purchaseBuilder => {
+                    purchaseBuilder.HasKey(purchase => purchase.Id);
+                    purchaseBuilder.HasOne(purchase => purchase.ItemEntity)
+                        .WithMany();
+                });
```

Comment:
Maybe rename purchaseBuilder to salesLineItemBuilder.

Reply:
This is because we have a seperate entity called Purchase and it was kinda confusing why we are building purchase from here at start. So renaming could help.

Reply:
> This is because we have a seperate entity called Purchase and it was kinda confusing why we are building purchase from here at start. So renaming could help.

I did it , i made that emoji cause i forgot to change name after pasting